blueprints/codes/eurocode/nen_en_1993_1_1_c2_a1_2016/chapter_6_ultimate_limit_state/formula_6_21.py

```diff
@@ -0,0 +1,74 @@
+"""Formula 6.21 from NEN-EN 1993-1-1+C2+A1:2016: Chapter 6 - Ultimate Limit State."""
+
+from blueprints.codes.eurocode.nen_en_1993_1_1_c2_a1_2016 import NEN_EN_1993_1_1_C2_A1_2016
+from blueprints.codes.formula import Formula
+from blueprints.codes.latex_formula import LatexFormula, latex_replace_symbols
+from blueprints.type_alias import MM2, MPA, N
+from blueprints.validations import raise_if_less_or_equal_to_zero, raise_if_negative
+
+
+class Form6Dot21ShearStressIOrHSection(Formula):
+    r"""Class representing formula 6.21 for the calculation of [$\tau_{Ed}$]."""
+
+    label = "6.21"
+    source_document = NEN_EN_1993_1_1_C2_A1_2016
+
+    def __init__(
+        self,
+        v_ed: N,
+        a_w: MM2,
+        a_f: MM2,
+    ) -> None:
+        r"""[$\tau_{Ed}$] Calculation of the design elastic shear stress [$MPa$].
+        For I- or H-sections the shear stress in the web may be taken with this equation.
+
+        NEN-EN 1993-1-1+C2+A1:2016 art.6.2.6(5) - Formula (6.21)
+
+        Parameters
+        ----------
+        v_ed : N
+            [$V_{Ed}$] Design shear force [$N$].
+        a_w : MM2
+            [$A_w$] Area of the web [$mm^2$].
+        a_f : MM2
+            [$A_f$] Area of one flange [$mm^2$].
+        """
+        super().__init__()
+        self.v_ed = v_ed
+        self.a_w = a_w
+        self.a_f = a_f
+
+    @staticmethod
+    def _evaluate(
+        v_ed: N,
+        a_w: MM2,
+        a_f: MM2,
+    ) -> MPA:
+        """Evaluates the formula, for more information see the __init__ method."""
+        raise_if_negative(v_ed=v_ed, a_f=a_f)
+        raise_if_less_or_equal_to_zero(a_w=a_w)
+        if a_f / a_w < 0.6:
+            raise ValueError("A_f / A_w must be greater than or equal to 0.6")
```

Comment:
maybe change to be more in the form as what is stated in the value error: 
```suggestion
        if not a_f / a_w >= 0.6:
            raise ValueError("A_f / A_w must be greater than or equal to 0.6")
```